### ISSUE TITLE
Is dataset online or local?

### ISSUE THREAD

**Issue by krNeko9t** [alibaba-yuanjing-aigclab/GeoLRM#8]
In src/dataset/gobjaverse.py, the DataModuleFromConfig class used:
return wds.WebLoader(self.datasets['train'], batch_size=self.batch_size, num_workers=self.num_workers, shuffle=False, sampler=sampler),
but the self.datasets['train'] is a instance of torch.utils.data.Dataset, is this the proper way to use it? I know this isn't about algorithms, but I was hoping you could help me out with training data. Thanks anyway!
